Cargo.toml

```diff
@@ -152,6 +152,8 @@ alloy-chains = "0.1.5"
 alloy-rlp = "0.3.3"
 solang-parser = "=0.3.3"
 
+reth-rpc-types = { git = "https://github.com/paradigmxyz/reth", package = "reth-rpc-types"}
```

Comment:
Note: the Engine API RPC types are now part of Alloy: https://github.com/alloy-rs/alloy/tree/main/crates/rpc-types-engine